packages/core/ios/Sources/DdSdk.m

```diff
@@ -24,4 +24,14 @@ @interface RCT_EXTERN_MODULE(DdSdk, NSObject)
                  withResolver:(RCTPromiseResolveBlock)resolve
                  withRejecter:(RCTPromiseRejectBlock)reject)
 
+RCT_EXTERN_METHOD(telemetryDebug:(NSString)message
+                 withResolver:(RCTPromiseResolveBlock)resolve
+                 withRejecter:(RCTPromiseRejectBlock)reject)
+
+RCT_EXTERN_METHOD(telemetryError:(NSString)message
+                 withStack:(NSString?)stack
+                 withKing:(NSString?)kind
```

Comment:
```suggestion
                 withKind:(NSString?)kind
```

Reply:
I made this typo 20 times while working on this, it looks like I can't type kind 😄